scrunch/datasets.py

```diff
@@ -2412,14 +2409,22 @@ def fork(self, description=None, name=None, is_published=False,
             **kwargs
         )
 
+        owner = kwargs.get("owner")
         if preserve_owner:
             body['owner'] = self.resource.body.owner
+        elif owner:
+            body["owner"] = (
+                owner if owner.startswith("http") else get_project(owner).url
```

Comment:
These should be encouraged to be Crunch objects rather than strings. For projects don't allow strings, urls or paths. Because you'll have problems validating here.

Have this be a Project instance.

Reply:
**question:** @jjdelc Double-checking on this - Isn't [get_project()](https://github.com/Crunch-io/scrunch/blob/master/scrunch/datasets.py#L180) validating the string for us? It will return a project instance if the path can be resolved, and raise an `InvalidPathError` it cannot:

```
InvalidPathError                          Traceback (most recent call last)
Cell In[8], line 7
      4 ygdc_crunch.Crunch()
      6 name_pr = "TechOps|Alex 3"
----> 7 pr = scrunch.get_project(name_pr)

File ~\AppData\Local\Continuum\anaconda2\envs\py311test\Lib\site-packages\scrunch\datasets.py:206, in get_project(project, connection)
    203 _project = Project(ret)
    205 if sub_project:
--> 206     _project = _project.get(sub_project)
    208 return _project

File ~\AppData\Local\Continuum\anaconda2\envs\py311test\Lib\site-packages\scrunch\datasets.py:552, in Project.get(self, path)
    550 for p_name in Path(path).get_parts():
    551     try:
--> 552         node = node.get_child(p_name)
    553     except KeyError:
    554         raise InvalidPathError('Project not found %s' % p_name)

File ~\AppData\Local\Continuum\anaconda2\envs\py311test\Lib\site-packages\scrunch\datasets.py:571, in Project.get_child(self, name)
    568         return Project(tup.entity)
    569     return self.root.dataset[name]
--> 571 raise InvalidPathError('Invalid path: %s' % name)

InvalidPathError: Invalid path: Alex 3
```

Reply:
Thanks @jjdelc - so a green light on allowing string paths?